packages/contentstack-export-to-csv/package.json

```diff
@@ -1,7 +1,7 @@
 {
   "name": "@contentstack/cli-cm-export-to-csv",
   "description": "Export entities to csv",
-  "version": "1.3.12",
+  "version": "1.3.13",
```

Comment:
@cs-raj This is a new feature enhancement and should be updated to a minor version.
@antonyagustine @shafeeqd959 This feature should be enabled from a specific CLI version as this can be a somewhat breaking change for customers who enabled automation with exported data. Hence this should restrict to a minor version update as discussed today.

Reply:
@netrajpatel I have added the minor version update for the cli-cm-export-to-csv